### ISSUE TITLE
False negatives for some unused features

### ISSUE THREAD

**Issue by nox** [rust-lang/rust#39996]
The two following feature attributes won't warn if there are no plugins no old-style custom derive plugins:

```rust
#![feature(custom_derive)]
#![feature(plugin)]
```

**Comment by keeperofdakeys** [comment]
cc @jseyfried Is this an issue?

**Comment by sanmai-NL** [comment]
Duplicate of #23528, @Mark-Simulacrum?

**Comment by Mark-Simulacrum** [comment]
Yeah, I'd say so. Thanks!
